recipes/onmail/webview.js

```diff
@@ -0,0 +1,23 @@
+const _path = _interopRequireDefault(require('path'));
+
+function _interopRequireDefault(obj) {
+  return obj && obj.__esModule ? obj : { default: obj };
+}
+
+module.exports = Ferdium => {
+  // TODO: If your onMail service has unread messages, uncomment these lines to implement the logic for updating the badges
+  const getMessages = () => {
+    let countImportant = 0;
+    const inboxLinks = document.querySelectorAll('p.truncate');
+    for (const label of inboxLinks){
+      if (label.textContent) {
+        countImportant = label.nextSibling.textContent;
```

Comment:
Please see the other recipes. It would be safe to use `Ferdium.safeParseInt()` here

Reply:
Also right, I didn't think it was needed, adding it...